extensions/variance/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Druid - a distributed column store.
```

Comment:
I know a lot of the pom's use this notice, but can you use the one present in the java files?